studio-docs/source/org/members.mdx

```diff
@@ -1,19 +1,46 @@
 ---
 title: Managing organization members
-sidebar_title: Members
+sidebar_title: Members and roles
 ---
 
 ## Inviting members
 
-Invite individual members by email address from your organization's Members tab.
+**Admins** can invite individual members by email address from your organization's Members tab.
 
-You can also create an **invite link** from your organization's Settings tab. Any number of individuals can use the same invite link.
+**Admins** can also create an *invite link* from your organization's Settings tab, which can be used and re-used multiple times for individuals to join the organization.
 
-> **Do not share your invite link publicly.** Anyone with the link can join your organization. If your invite link becomes compromised, you can replace or disable it from the Settings tab.
+> *Do not share your invite link publicly.* Anyone with the link can join your organization. If your invite link becomes compromised, you can replace or disable it from the Settings tab.
+
+If your organization has user roles enabled, **Admins** will be able to control which role is attached to each invitation and which role your invitation link is for.
 
 ## Removing members
 
-Remove members from your organization's Members tab.
+**Billing Managers** and **Admins** can remove members from your organization from your organization's Members tab.
+
+## User roles
+
+> User roles is currently an invite-only capability in Studio.
+Please contact support@apollographql.com if you're interested in access for your organization.
```

Comment:
We discussed this in Slack and landed on describing this as request-only rather than Enterprise, and putting the burden on Support: https://meteor.slack.com/archives/C016EN3L91A/p1597960602003800. This will help us gauge interest, and we can change the wording later if the Support burden becomes to high.